--- a/toolchain/config.bzl
+++ b/toolchain/config.bzl
@@ -1,4 +1,6 @@
 load("@bazel_tools//tools/build_defs/cc:action_names.bzl", "ACTION_NAMES")
 load("@rules_cc//cc:cc_toolchain_config_lib.bzl", "action_config", "feature", "flag_group", "flag_set")
+load("@rules_cc//cc/private/toolchain_config:cc_toolchain_config_info.bzl", "CcToolchainConfigInfo")
+load("@rules_cc//cc/private:cc_common.bzl", "cc_common")

 def _tool_path(bins, toolchain_prefix, tool_name):
